Modules/_bisectmodule.c

```diff
@@ -66,7 +66,7 @@ internal_bisect_right(PyObject *list, PyObject *item, Py_ssize_t lo, Py_ssize_t
     if (sq_item == NULL) {
         return -1;
     }
-    if (Py_EnterRecursiveCall("in _bisect.bisect_right") < 0) {
+    if (Py_EnterRecursiveCall("in _bisect.bisect_right")) {
```

Comment:
This is not covered in my PR. I think it’s needed.

Reply:
Yes, it is. See also https://github.com/python/cpython/issues/101965#issuecomment-1437592496